Api/Application/Services/EmailNotifier.cs

```diff
@@ -0,0 +1,75 @@
+using Api.Domain;
+
+using MailKit.Net.Smtp;
+using MailKit.Security;
+
+using Microsoft.IdentityModel.Tokens;
+
+using MimeKit;
+using MimeKit.Text;
+
+public class EmailNotifier : INotifier<EmailContent>
+{
+    private string _smtpServer = string.Empty;
+    private string _smtpUser = string.Empty;
+    private string _smtpPassword = string.Empty;
+    private int _smtpPort = 587;
+
+    public EmailNotifier(IConfiguration configuration)
+    {
+        ConfigSmtpServer(configuration);
+    }
+
+    public void Notify(EmailContent emailContent)
```

Comment:
It is not better for this method to be asynchronous.

Reply:
I guess not, based on the documentation that I reviewed for this SMTP service

Reply:
Its okey